File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/wheelmanager/WheelManagerSnapshot.java
package org.firstinspires.ftc.teamcode.wheelmanager;

public class WheelManagerSnapshot {
    public final double time;
    public final int[] encoderPositions;

    public WheelManagerSnapshot(double time, int[] encoderPositions) {
        this.time = time;
        this.encoderPositions = encoderPositions;
    }
}

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/wheelmanager/WheelManagerSnapshot.kt
package org.firstinspires.ftc.teamcode.wheelmanager

import org.firstinspires.ftc.teamcode.units.Degree
import org.firstinspires.ftc.teamcode.units.Distance

data class WheelManagerSnapshot(
        val time: Double,
        val encoders: IntArray,
        val distance: Distance,
        val degrees: Degree
)
